command.go

```diff
@@ -102,6 +102,17 @@ type CommandFunc struct {
 	// See Command for details about the accepted signatures.
 	Func interface{}
 
+	// An optional usage string for this function. If set, then this replaces
+	// the default one that shows the types (but not names) of arguments.
+	Usage string
+
+	// Set of options to not set from the environment more user-friendly-syntax
+	IgnoreEnvOptions []string
```

Comment:
TBH, I think we should just permit one or the other of these two arguments, and not both. In the worst case we have to make one pass through to construct a map; I don't think that adds that much overhead that it's inefficient. Or, force users to use the map[string]struct{}

Reply:
This change is to address Achille's suggestion on https://github.com/segmentio/cli/pull/26#discussion_r740625480

I am happy to make changes as y'all think it fits/works the best as I can see both ways fit 🙂